Automattic/wp-calypso
# Jetpack connect: add a feature flag to allow disabling site questions

#### Changes proposed in this Pull Request

* Put site questions in Jetpack connect flow behind a feature flag on all environments; the flow should stay visible everywhere after this change.
* ~Disable related E2E tests (convo p1562574334001800-slack-e2e-testing-discuss)~

Next up we'd flip these feature flags off to disable the steps in all environments. (PR: https://github.com/Automattic/wp-calypso/pull/34620)

Context paObgF-k1-p2

<img src="https://user-images.githubusercontent.com/87168/60793477-743bdc80-a170-11e9-8882-21b8b3c3573b.png" alt="Jetpack connect steps" width="450" />


#### Testing instructions
- Create a new Jetpack site and connect it to .com using localhost.
- Can still see extra questions?
